test/new-e2e/tests/containers/kindvm_test.go

```diff
@@ -8,27 +8,37 @@ package containers
 import (
 	"context"
 	"fmt"
-	"os"
-	"path"
-	"testing"
-
 	fakeintake "github.com/DataDog/datadog-agent/test/fakeintake/client"
 	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/runner"
 	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/utils/infra"
 	"github.com/DataDog/test-infra-definitions/scenarios/aws/kindvm"
-
 	"github.com/pulumi/pulumi/sdk/v3/go/auto"
 	"github.com/stretchr/testify/suite"
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/tools/clientcmd"
+	"os"
+	"path"
+	"strings"
+	"testing"
 )
 
 type kindSuite struct {
 	k8sSuite
+	k8sVersion string
+	stackName  string
 }
 
 func TestKindSuite(t *testing.T) {
-	suite.Run(t, &kindSuite{})
+
+	KubeVersions := []string{"1.27", "1.23", "1.20", "1.19"}
```

Comment:
```suggestion
	kubeVersions := []string{"1.27", "1.23", "1.20", "1.19"}
```
Small suggestion. I think it could also be hardcoded as a constant somewhere else